crates/noirc_evaluator/src/ssa/code_gen.rs

```diff
@@ -199,6 +199,20 @@ impl<'a> IRGenerator<'a> {
         &self.context.context.def_interner
     }
 
+    fn evaluate_prefix_expression(
+        &mut self,
+        rhs: NodeId,
+        op: HirUnaryOp,
+    ) -> Result<NodeId, RuntimeError> {
+        let rtype = self.context.get_object_type(rhs);
+        match op {
+            HirUnaryOp::Minus => todo!(), //todo: Ok(self.context.new_instruction(0, rhs, Operation::Sub, rtype))
```

Comment:
dead comment

Reply:
The comment is here to indicate how to implement this, but it should be tested so I leave this for another PR.

Reply:
I could implement it finally.